monitor.py

```diff
@@ -6,18 +6,20 @@
 import requests
 from datetime import datetime
 import xml.etree.ElementTree as ET
+import monitoring_config
```

Comment:
```suggestion
import constants
```
Please rename it to constants itself.